src/components/ebay-pagination/index.js

```diff
@@ -65,6 +65,7 @@ function getTemplateData(state) {
 
 function init() {
     this.pageContainerEl = this.el.querySelector('.pagination__items');
+    this.pageContainerEl.style.flexWrap = 'nowrap';
```

Comment:
Any particular reason this is here rather than in CSS/HTML?

Reply:
@seangates told me that skin had to wrap and hide elements that wrap using the overflow of the container.  Using wrap messes with the math, so I'm overriding it in ebayui as a progressive enhancement.